### ISSUE TITLE
Duplicate rules

### ISSUE THREAD

**Issue by atomoc** [Jako/multiTV#127]
If multitv type values more than one there is a duplication of the rules:
![2015-02-19 12 53 21](https://cloud.githubusercontent.com/assets/465108/6264603/4e774b92-b836-11e4-9442-cb03099dd0f5.png)

**Comment by atomoc** [comment]
Can you solve this problem by function include_once?

**Comment by Jako** [comment]
Yes, the CSS is referenced multiple times. MODX limitation, could be solved somehow.
